tox.ini

```
@@ -19,10 +17,8 @@ commands = coverage run --parallel-mode --source django_filters ./runtests.py --
 setenv =
        PYTHONDONTWRITEBYTECODE=1
 deps =
-    django42: Django>=4.2,<5.0
-    django50: Django>=5.0,<5.1
-    django51: Django>=5.1,<5.2
-    django52: Django>=5.2a1,<6.0
+    django52: Django>=5.2,<6.0
+    django60: Django>=6.0a1,<7.0
```

Shouldn't this have been 6.1 instead of 7.0?

```suggestion
    django60: Django>=6.0a1,<6.1
```